### DISCUSSION TITLE
The poll is sending and triggering the same POLL.VOTE every 10 minutes

### DISCUSSION THREAD

**Discussion by TitoBahe** [devlikeapro/waha#652] [Q&A] [answered]
I am receiving the poll.vote event every 10 minutes, even when the user hasn’t changed his vote.
Is there a way of only receiving the poll.vote event when somebody actually votes?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
Hi! Can not reproduce it - I've tried DM poll, is it the case you're talking about (not group)?

Could you validate the following:
1. You're using WAHA latest version (2024.11.6 or later)
2. The `id` field in POST body is different (in format `evt_XXXXX`)
3. is `payload.vote.id` is different
4. The `x-webhook-request-id` HTTP header is different for both request
5. Is there any errors may be in logs for this poll/vote/session?


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by TitoBahe** [reply]
**1. You're using WAHA latest version (2024.11.6 or later):**
**R-** Yes, I think is the latest. It was created 9 days ago, it says here. At least. 

**2. The id field in POST body is different (in format evt_XXXXX):** 
**R-** No, they are not different:  
**10 min ago** : evt_01jgm5z49bagcr9hnkr4sp21r8
**Now**: evt_01jgm5z49bagcr9hnkr4sp21r8

**3. is payload.vote.id is different?:**
**R-** No, they are not different: 

**10 min ago** : false_40723973035@c.us_C9C9D7D97F91E5D441A40F4DAF1725DC
**Now**: false_40723973035@c.us_C9C9D7D97F91E5D441A40F4DAF1725DC

**4. The x-webhook-request-id HTTP header is different for both request?:**
**R-** They are the same:

**10 min ago** : 01JGM5Z49BC009HDK6E4XATY4R
**Now**: 01JGM5Z49BC009HDK6E4XATY4R

**5. Is there any errors may be in logs for this poll/vote/session?** 
**R-** There are no erros at all. They are working normally. 

6. And theres another problem. When I send a poll to a brazilian number, it arrives well, they actually can answer the poll normally, they see the poll and eveything. But, for me it doesnt appear on whatsapp, so I cant see the poll I just sent. 
This problem doesnt happen with my Romanian number. Only my Brazilian one. 

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by devlikepro** [reply, marked as answer]
ok, so it means it's not duplicated webhook event, but your app responded with an error and it keep retrying sending the event to your webhook endpoing
Try disabling the retries - on dashboard or when your start a new session via config

https://waha.devlike.pro/docs/how-to/webhooks/#retries
```json
{
  "name": "default",
  "config": {
    "webhooks": [
      {
        "url": "https://webhook.site/11111111-1111-1111-1111-11111111",
        "events": [
          "message"
        ],
        "retries": {
          "delaySeconds": 1,
          "attempts": 1,
          "policy": "constant"
        }
      }
    ]
  }
}

```


[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Reply by TitoBahe** [reply]
I have a fallback logic to resend the poll automatically in case it fails, will the be desabled also since the attempts will be set to 1?
I mean, that is only triggered by the poll.fail event.

If you dont mind, sorry for bothering you so frequently, I have some doubts:

1. I think I didnt actually understand the problem. If I had an error, should the poll.fail event be treggered? So I am having an error on the poll.vote event but is not being translated into a poll.failt event?

2. What exactly will be affected when the retries attempts are set to 1, I mean, when I implement what you told me above?
I mean, these are the events I use:
"events": [
                        "message.any", 
                        "session.status", 
                        "poll.vote.failed",
                        "poll.vote", 
                        "group.join", 
                        "message.reaction"
                    ]
And will the send Menssages, send Video, audio or files be affected somehow?

[![patron:PLUS](https://img.shields.io/badge/patron-PLUS-a0e6ba)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)

**Comment by devlikepro** [comment]
6. Just to make sure it's not infrastructure/webhook handler/retry issues - could you validate it via https://webhook.site/ ? You can copy the URL and use it as webhook to receive and observe data

[![patron:PRO](https://img.shields.io/badge/patron-PRO-188a42)](https://waha.devlike.pro/docs/how-to/plus-version/#tiers)
